wiki/People/The_Team/Account_support_team/ru.md

```diff
@@ -0,0 +1,50 @@
+---
+tags:
+  - техподдержка
+  - служба поддержки
+---
+
+# Служба техподдержки
+
+*Эта статья – о службе поддержки, отвечающей на электронные письма по адресам `@ppy.sh`. О группе пользователей, помогающих на форумах, см. [Команда поддержки](/wiki/People/The_Team/Support_Team).*
```

Comment:
it's a [hatnote from the ASC](https://osu.ppy.sh/wiki/ru/Article_styling_criteria/Formatting#%D0%BF%D1%80%D0%B5%D0%B4%D0%BC%D0%B5%D1%82-%D1%81%D1%82%D0%B0%D1%82%D1%8C%D0%B8) saying there's another article about a similar topic, so it doesn't have to be in the original. it's widely [used in Russian Wikipedia](https://ru.wikipedia.org/wiki/%D0%A8%D0%B0%D0%B1%D0%BB%D0%BE%D0%BD:%D0%AD%D1%82%D0%B0_%D1%81%D1%82%D0%B0%D1%82%D1%8C%D1%8F), which served as an example for the ASC in this case

Reply:
it doesn't have to be exact same but I'm asking why not. what about this sentence makes it relevant to russian speakers and not anyone else?

Reply:
does the hatnote need an extra sentence to clarify what the article is, and doesn't the lead paragraph make it clear already? i guess it would be a style thing, though from looking at machine translation it seems a little redundant

Reply:
the two-sentence structure makes it a proper explanation which doesn't read incomplete and choppy and essentially says `this is <...>, not to be confused with <...> .`, very briefly explaining why. chopping it down to a single sentence makes it no different than an ambiguity-denoting hatnote (which leads to a disambiguation page for the term) which I would really want to avoid here for it's a completely different matter, so [when I was translating ASC,] I went ahead and used the best practices available out there on the internet*s*